packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: url_launcher_platform_interface
+description: A common platform interface for the url_launcher plugin.
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
+version: 0.1.0
```

Comment:
Let's add a comment reminding that is really bad to make a breaking change to this package, and that it's usually better to compromise on a less clean platform interface API. You can refer to the relevant section here: https://docs.google.com/document/d/1LD7QjmzJZLCopUrFAAE98wOUQpjmguyGTN2wd_89Srs/edit#heading=h.xabf9lffxrkb (I'll add a go link for that bookmark)

Reply:
https://github.com/flutter/website/pull/3142 adds https://flutter.dev/go/platform-interface-breaking-changes

Reply:
Done (in the README)